### DISCUSSION TITLE
starstruck achievement

### DISCUSSION THREAD

**Discussion by XakimovNosirbek** [community/community#20627] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello! I've noticed something is not okay with the starstruck achievement. even though i have a repository with 17 stars i am not getting achievement.
<img width="1066" alt="starstark" src="https://user-images.githubusercontent.com/68727555/179310225-81199e3a-a5ce-47fc-af70-74e6f993a384.png">

**Comment by radar-blip** [comment, marked as answer]
But you did. 👍🏼

**Comment by devkennyy** [comment]
> even though i have a repository with 17 stars i am not getting achievement.

The Starstruck achievement is the first one on your list. Hover over it for more information.
